packet/flatcar-linux/kubernetes/ssh.tf

```diff
@@ -90,6 +90,11 @@ resource "null_resource" "bootkube-start" {
     destination = "$HOME/assets/manifests-networking/calico-policy.yaml"
   }
 
+  provisioner "file" {
+    source     = "${path.module}/calico/host-endpoint-controller.yaml"
+    destination = "$HOME/assets/manifests-networking/host-endpoint-controller.yaml"
+  }
```

Comment:
This is applied on cluster creation and never applied again? Not sure what happened to resources created there

Reply:
terraform render bootkube handles it. IIRC they stay there and don't get deleted, which is also nice so you can see what got applied in or how it rendered there when you want to troubleshoot.

Reply:
Oh, right. Thanks!